app/component/toast.tsx

```diff
@@ -0,0 +1,33 @@
+import Failed from "react:~assets/failed.svg"
+import Submit from "react:~assets/submit.svg"
```

Comment:
nit: Maybe call it `sent` or `pending`?

Reply:
[rename submit to sent](https://github.com/pilagod/waallet-extension/pull/158/commits/0f11094603dab5c6e9b19ba51c101c1aef426213)